### DISCUSSION TITLE
OpenWRT GUI inaccessible once VPN is started

### DISCUSSION THREAD

**Discussion by sgtjamez** [tteck/Proxmox#3364] [Help Needed] [answered]
Forgive me if you've seen this issue posted by me elsewhere. I am desperately trying to resolve this issue.
I am able to get the OpenWRT VM up and running, and once I go through the VPN configuration process everything is ok until enable and start the VM. I get a message about unsaved changes and as soon as I click save and apply the countdown spinner starts and that;s it. The GUI is lost. I can still access OpenWRT from the VM console in Proxmox. Also the VPN is working because a Debian LXC running through the LAN vmbr shows an IP from my VPN service when tested with `# curl ifconfif.co`.

I can go back into the openvpn config and set enable to '0' and then reboot the VM. Boom... the GUI is back.  

Please, in the name of my cat Mittens, someone tell me they know how to fix this!!

Please and thank you!!

**Comment by gladiatormax** [comment, marked as answer]
Hi,
For the love off Mittens, follow this guide,
https://youtu.be/3mPbrunpjpk?si=y9_XV54loZTQeLIe
He also explains haw to access you gui.

**Reply by sgtjamez** [reply]
Mittens and I wholeheartedly thank you for your reply. That's the exact video that got me started on this. Turns out the issue is my own fault. My laptop and PC are on vlan 10 and the OpenWRT server is on vlan 40. Once I changed my laptop/PC to vlan 40, I had access. Kinda dumb on my part. I have some basic info from another post on how to set vlan 10 to have access but I'm still getting to that after I resolve the NFS share issues that followed... See related post if your're still feeling helpful.

-Jim
